api/genomics/docs/unsorted_vcf.md

```diff
@@ -0,0 +1,31 @@
+## Answering some questions on how our tools interact with unsorted VCF files
+
+### Does Plink require a VCF file to be sorted?
+No, but another flag `--make-bed` needs to be added to the binary file creation step.
```

Comment:
One thing to understand here is whether we are just pushing the sorting burden to later in the flow. Presumably `--make-bed` is actually sorting the variants, and therefore needs to do the same work as the input VCF.

Reply:
I couldn't find documentation that explicitly stated this but that's almost certainly the case.